eftest/src/eftest/runner.clj

```diff
@@ -182,7 +191,10 @@
        (binding [report/*context* (atom {})
                  test/report      (:report opts progress/report)]
          (test/do-report {:type :begin-test-run, :count (count vars)})
-         (let [counters (test-all vars opts)
+         (let [executor (threadpool-executor opts)
```

Comment:
We only need  an executor if we're executing threaded tests, otherwise it can be `nil`.